index.html

```diff
@@ -0,0 +1,374 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Aclonica&family=Acme&family=Actor&display=swap"
+          rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>SCOUT</title>
+</head>
+<body>
+<header>
+    <div class="headerContainer">
+        <div class="logo">
+            <a href="#"><img src="image/SKOUT.png" alt="SKOUT"></a>
+        </div>
+        <div class="headerMenu">
+            <div class="menuIcon">
+                <span></span>
+            </div>
+            <nav>
+                <ul>
+                    <li>
+                        <a href="#">Why Skout</a>
+                    </li>
+                    <li>
+                        <a href="#">Products</a>
+                    </li>
+                    <li>
+                        <a href="#">Partners</a>
+                    </li>
+                    <li><a href="#">Company</a></li>
+                    <li><a class="current" href="#">Tips & Tools</a>
+                        <hr>
```

Comment:
![Border line](https://user-images.githubusercontent.com/73053424/120236011-f56c5300-c263-11eb-9a00-0c1348c9bd7d.png)
If I make the underline as a block border, it will be too large and will not fit the job.

Reply:
yes, agree, use pseudoelement ::before or after here
and please don't raise your voice to me anymore)